packages/search-input/src/SearchInput/SearchInput.styles.ts

```diff
@@ -9,48 +9,129 @@ import {
+  outline: none;
+`;
+
+export const inputWrapperStyle = css`
+  position: relative;
+  display: grid;
+  grid-auto-flow: column;
+  align-items: center;
+  justify-items: center;
+  border: 1px solid;
+  border-radius: 6px;
+  z-index: 0;
+  transition: ${transitionDuration.default}ms ease-in-out;
+  transition-property: border-color, box-shadow;
+`;
+
+export const inputWrapperSizeStyle: Record<SizeVariant, string> = {
   [SizeVariant.Small]: css`
     font-size: ${typeScales.body1.fontSize}px;
     line-height: ${typeScales.body1.lineHeight}px;
+    height: 28px;
+    grid-template-columns: 28px 1fr;
+    grid-auto-columns: 28px;
   `,
   [SizeVariant.Default]: css`
     font-size: ${typeScales.body1.fontSize}px;
     line-height: ${typeScales.body1.lineHeight}px;
+    height: 36px;
+    grid-template-columns: 36px 1fr;
+    grid-auto-columns: 36px;
   `,
   [SizeVariant.Large]: css`
     font-size: 18px;
     line-height: 32px;
+    height: 48px;
+    grid-template-columns: 48px 1fr;
+    grid-auto-columns: 48px;
   `,
 };
 
-export const inputContainerStyle = css`
-  position: relative;
-  display: flex;
-  align-items: center;
-  z-index: 0;
-`;
+export const inputWrapperThemeStyle: Record<Theme, string> = {
+  [Theme.Light]: css`
+    color: ${palette.black};
+    background: ${palette.white};
+    border-color: ${palette.gray.base};
+  `,
+  [Theme.Dark]: css`
+    color: ${palette.gray.light2};
+    background-color: ${palette.gray.dark4};
+    border-color: ${palette.gray.base};
+  `,
+};
+
+export const inputWrapperInteractiveStyles: Record<Theme, string> = {
+  [Theme.Light]: css`
+    &:hover,
+    &:active {
+      &:not(:disabled):not(:focus-within) {
+        box-shadow: ${hoverRing.light.gray};
+      }
+    }
+  `,
+  [Theme.Dark]: css`
+    &:hover,
+    &:active {
+      &:not(:disabled):not(:focus-within) {
+        box-shadow: ${hoverRing.dark.gray};
+      }
+    }
+  `,
+};
+
+export const inputWrapperFocusStyles: Record<Theme, string> = {
+  [Theme.Light]: css`
+    &:not(:disabled):focus-within {
+      box-shadow: ${focusRing.light.input};
+      border-color: ${palette.white};
+    }
+  `,
+  [Theme.Dark]: css`
+    &:not(:disabled):focus-within {
+      box-shadow: ${focusRing.dark.input};
+      border-color: ${palette.gray.dark4};
+    }
+  `,
+};
+
+export const inputWrapperDisabledStyle: Record<Theme, string> = {
+  [Theme.Light]: css`
+    cursor: not-allowed;
+    color: ${palette.gray.base};
+    background-color: ${palette.gray.light2};
+    border-color: ${palette.gray.light1};
+  `,
+  [Theme.Dark]: css`
+    cursor: not-allowed;
+    color: ${palette.gray.dark2};
+    background-color: ${palette.gray.dark3};
+    border-color: ${palette.gray.dark2};
+  `,
+};
 
 export const baseInputStyle = css`
   font-size: inherit;
   line-height: inherit;
+  color: inherit;
+  background-color: inherit;
   font-family: ${fontFamilies.default};
   width: 100%;
-  height: 36px;
+  height: 1.5em;
```

Comment:
Should we be preferring `em`? Should this be in our style guide?

Reply:
I would advocate for `em` for most things, yeah, since it makes resizing easier